src/binary-reader-objdump.cc

```diff
@@ -897,7 +897,7 @@ Result BinaryReaderObjdumpDisassemble::OnOpcodeType(Type type) {
   if (!in_function_body) {
     return Result::Ok;
   }
-  if (current_opcode == Opcode::SelectT) {
+  if (current_opcode == Opcode::SelectT || current_opcode == Opcode::CallRef) {
```

Comment:
Maybe more opcodes could go here (ref.null?). This is confusing for me, the two toString variants works differently, although they could be merged.